### ISSUE TITLE
Mystery Weight in inventory

### ISSUE THREAD

**Issue by Chaps-Fantastic** [ForgottenGlory/Living-Skyrim-3#635]
**LS Version**
3.7.0

**Describe the bug**
A There are 223 units of weight added to my inventory. The screencap shows that I have removed everything from my inventory except for quest items. This should show a weight of 0, but it is showing at 223. 

I noticed this after I completed Stalleo's "Infiltration" quest at Treva's Watch.
**Save File**
[Save284_BB9A0BC3_0_4768756C2D566569207468652046726F67746F6E677565_DBMDGSafehouse_010109_20220117060025_14_1.zip](https://github.com/ForgottenGlory/Living-Skyrim-3/files/7879335/Save284_BB9A0BC3_0_4768756C2D566569207468652046726F67746F6E677565_DBMDGSafehouse_010109_20220117060025_14_1.zip)

**Screenshots**
![Screenshot (Weight issue)](https://user-images.githubusercontent.com/82067609/149716053-8fea8b15-4913-40f0-98d4-5b869e0f7aec.png)

**Additional context**

**Comment by Chaps-Fantastic** [comment]
A user suggested using the "player.showinventory" command and then the "player.removeitem (item ID)"  command to remove every bugged item. This showed the following food items...
![Inventory](https://user-images.githubusercontent.com/82067609/149745397-b5f1840c-45b6-425d-b94d-cecae3a7c83f.jpg)

I removed those items using the command above and it then dropped the weight down to the 0 it should have been.
![Capture](https://user-images.githubusercontent.com/82067609/149745665-b90cc5fb-7bbc-417f-8574-69bc57fc82ea.JPG)

Once I saw what it was, I’m almost positive this is a LotD Craftloot issue. Recently, I had been cooking food and there must’ve been an issue with the return to storage aspect after I was done.
